File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> res = new HashMap<Integer,Integer>();
        int result[] = new int[2];
        for(int i=0;i<nums.length;i++){
            res.put(nums[i],i);
        }
        for(int i=0;i<nums.length;i++){
            if(res.containsKey(target-nums[i]) && res.get(target-nums[i]) != i){
                result[0] = i;
                result[1] =  res.get(target-nums[i]);
                return result;
            }
        }
        return result;
        
    }
}

File: Data Structures & Algorithms/sliding-window-maximum/submission-1.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        if k > len(nums):
            return []
        
        i = 0
        j = k-1
        result = []
        while j<len(nums):
            max_value = max(nums[i:j+1])
            result.append(max_value)
            i += 1
            j += 1

        return result  
        